server/src/test/java/io/spine/examples/shareaware/server/investment/given/InvestmentTestEnv.java

```diff
@@ -347,6 +348,22 @@ public static InvestmentView investmentViewAfter(PurchaseShares firstPurchase,
                 .vBuild();
     }
 
+    public static WalletBalance walletBalanceAfter(PurchaseShares firstPurchase,
+                                                   PurchaseShares secondPurchase,
+                                                   Wallet wallet) {
+        Money firstPurchasePrice = multiply(firstPurchase.getSharePrice(),
```

Comment:
See? It would be much more convenient to have a mixin interface for `PurchaseShares` which would calculate the price in its `default` method.